invenio_search_ui/bundles.py

```diff
@@ -42,12 +42,13 @@
 js = NpmBundle(
     'js/invenio_search_ui/app.js',
     filters='requirejs',
-    depends=('node_modules/invenio-search-js/dist/*.js', ),
+    depends=('node_modules/invenio-search-js/dist/*.js', 'node_modules/d3/*'),
     output='gen/search.%(version)s.js',
     npm={
         "almond": "~0.3.1",
         'angular': '~1.4.10',
         'angular-loading-bar': '~0.9.0',
-        'invenio-search-js': '~0.1.6'
+        'd3': '^3.5.17',
```

Comment:
OK from my side - we're building the search bundle ourselves in order to properly exclude modules already loaded so won't have any service impact. We should however review how we create bundles so that we don't have e.g. angular or d3 included twice. E.g. the CSV file previewer is also using D3.